code/restful-api/restful-api/Models/Local.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+namespace RestfulApi.Models
+{
+    public class Local
```

Comment:
I think it makes more sense if `Local` has an `Endereco`, and not otherwise.

What do you think?

Reply:
Feito.